prime-router/build.gradle.kts

```diff
@@ -537,10 +543,20 @@ tasks.azureFunctionsPackage {
     finalizedBy("copyAzureScripts")
 }
 
+// copy metadata to main resources
```

Comment:
This is to leverage gradle resources provision practice - copying metadata from source tree to build directory - since we switched to URI (resources) as schema references.

this is to avoid change the current source tree structures:

<local-repo-base>
             |
             |----------> metadata
             |                           |
             |                           |--------> fhir_transforms
             |                           |
             |                           |--------> hl7_mapping
             |                           |-------------> other sub dirs
             | 
             |--------- src
             |                  |
             |                  |----------> main
             |                  |                       |
             |                  |                       |--------> resources
             |                  |                       .......

 the git mv approach is OK, since the metadata hl7_mapping, fhir_transforms will be copied to build area by convention, but the un-desirable is that it break the top level metadata folders original purpose - keep the schemas etc. in one place, now hl7_mapping, fhir_transforms are moved out, RS "metadata" are scattered around in multiple locations in source tree, I think it reduces the managibility.

later on if RS adding more "metadata", where should they go? go to top level metadata dir? or go into src/main/resources/metadata?

Reply:
Can we not just put them in a directory called metadata?  I'm not sure anything else has to change.

Reply:
"Can we not just put them in a directory called metadata? I'm not sure anything else has to change."

Can do that.